sqlparser/src/Token.sk

```diff
@@ -242,7 +242,7 @@ base class Token uses Equality, Show, Hashable {
   | TInteger _ -> "INTEGER"
   | THexInteger _ -> "INTEGER"
   | TFloat _ -> "FLOAT"
-  | TSingleQuotedString _ -> "STRING"
```

Comment:
This is not required and probably not what we want, as "STRING" here refers to the token type of a single quoted string in a SQL statement (such as `SELECT 'hello'`).

Reply:
However we probably want to get rid of [sqlparser/src/Parser.sk#L100](https://github.com/SkipLabs/skdb/blob/dd193e8e34e5cf59fb4eafd36333c450a22bb6ee/sqlparser/src/Parser.sk#L100):
```
    if (names.any(s -> s.lower.contains("int"))) {
      INTEGER()
    } else if (
      names.any(s ->
        s.lower.contains("char") ||
          s.lower.contains("clob") ||
          s.lower.contains("text") ||
-         s.lower.contains("string") // NOTE: This differs from SQLite.
      )
    ) {
      TEXT()
```

(which also makes me realize there is a typo two lines above, where `clob` should be `blob`).

Reply:
Ahh, I was wondering why I couldn't find lexing code for a keyword "STRING"!

Will fix both things.

Reply:
Actually, the `"clob"` thing was correct, my bad (cf. [SQLite's algorithm](https://www.sqlite.org/datatype3.html)).